lib/content-types/merge.js

```diff
@@ -60,6 +67,13 @@ const squish = (types) => {
         plugin.id = attribute.id;
         plugin.type = attribute.type;
 
+        // Set Default Label from Plugin Name
+        if (Object.keys(plugin.inputs).length === 1) {
+          const input = Object.keys(plugin.inputs)[0];
+
+          plugin.inputs[input].label = plugin.name;
```

Comment:
This is the merge. If someone configured an input plugin, and there is only one input, the name they give the configuration should be used as the label if one isn't passed.

```yaml
name: Foo
attributes:
  - name: My Awesome Text Input
    type: text
```

That'll result in `My Awesome Text Input` being the label. You'll notice on line 62 that the plugin name is overridden with the attribute name.